@navikt/core/react/src/tabs/parts/tab/Tab.tsx

```diff
@@ -0,0 +1,92 @@
+import cl from "clsx";
+import React, { forwardRef } from "react";
+import { BodyShort } from "../../../typography";
+import { OverridableComponent } from "../../../util";
+import { useTabsContext } from "../../Tabs.context";
+import { useTab } from "./useTab";
+
+export interface TabProps
+  extends Omit<React.ButtonHTMLAttributes<HTMLButtonElement>, "children"> {
+  /**
+   * Tab label.
+   */
+  label?: React.ReactNode;
+  /**
+   * Tab Icon.
+   */
+  icon?: React.ReactNode;
+  /**
+   * Value for state-handling.
+   */
+  value: string;
+  /**
+   * Overrides auto-generated id.
+   * @note Tab will generate an id if not provided. If you need to override it,
+   * make sure to also include the correct `aria-controls` id for the TabPanel it controls.
```

Comment:
`@note` er vel ikke gyldig JSDOC. Tror uansett ikke det blir så pent i prop-lista på Aksel. Kan ev. bruke markdown for å fremheve det.

Reply:
Fikset 🚀 
![Screenshot 2024-04-10 at 12 44 03](https://github.com/navikt/aksel/assets/26967723/6e5293c7-7481-4099-9618-200a844c484e)

Reply:
Er det forskjell på å ha ett linjeskift vs to? 🤔

Reply:
Testet nå, gjør ikke noen forskjell.